src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -1118,6 +1118,24 @@ static LinkAnimationHeader* D_80853D4C[][3] = {
       &gPlayerAnim_link_fighter_Rside_jump_endR },
 };
 
+// TODO: Figure out what the duplicate types mean when `curRoom.behaviorType2` is more understood
+typedef enum SpecialIdleType {
+    /* 0x00 */ SPECIAL_IDLE_LOOK_AROUND,
+    /* 0x01 */ SPECIAL_IDLE_COLD,
+    /* 0x02 */ SPECIAL_IDLE_HOT,
```

Comment:
HOT_1 and STRETCH_1 to be more consistent with the others?

Reply:
I see the numbers as temporary until we have better names. I would rather there be just "hot" instead of it also needing to be numbered

Reply:
why is this one special though compared to the others? If anything HOT_1 more clearly marks it as temporary

Reply:
Im assuming that this is the "real" hot one and the numbered ones will have a different name eventually. This is just conjecture.
The names of the environmental ones also do not matter and will not be referenced at all, only the common ones are seen in code.

Reply:
well these could be used to comment sSpecialIdleAnimations and maybe sSpecialIdleAnimSfxTypes too

based on grepping room files (e.g. `rg 'SCENE_CMD_ROOM_BEHAVIOR\(\w+, 0x02' extracted/ntsc-1.0/ -l`):

type 0x02 is used in "warm" rooms (gerudo area, most of fire temple, tower collapse exterior?)
type 0x03 is used in rooms with heat timers (a few fire temple rooms, DMC, volvagia)
type 0x04: other indoors, ZD, goron city (safe areas?)
type 0x05: link's house, kokiri houses, carpenter tent, carpenter boss's house in kak (houses?)
type 0x06: only back alley (day and night)

Not really sure what to make of the stretch animations (I can't view them easily because z64utils doesn't work for me) but it seems like HOT isn't the "real" hot animation

Reply:
just looking at the pointer data, warm areas vs hot areas should just use the same animation; the difference is that type three enables the heat timer. The same is the case for the various stretch animations, except that there's no unique behavior tied to them afaik.

Reply:
I see thanks, in that case I don't think HOT_1/STRETCH_1 should be treated as more important here

Reply:
Alright, I can work with that data to make some better names. I just didnt want to go through the work of getting all the room types rn lol

Reply:
I agree with the STRETCH_1 case, but in `HOT`'s case arguably one should be `HOT` and the other should be `VERY_HOT` even though they are the same animation because of how enum `RoomBehaviorType2` directly maps to `SpecialIdleType`.

Arguably, they could even just be defined as one enumeration, except it doesn't quite make sense to use RoomBehaviorType2 to values beyond 6

Reply:
Yeah I dont think they should be the same enum.
I think warm vs hot sounds fine instead of hot and very hot,